bundles/admin/donationProcessing/readDonations.tsx

```diff
@@ -106,8 +157,41 @@ export default React.memo(function ReadDonations() {
             {sortedDonations.map((donation: any) => {
               const donor = donors?.find((d: any) => d.pk === donation.donor);
               const donorLabel = donor?.alias ? `${donor.alias}#${donor.alias_num}` : '(Anonymous)';
+              const donobids = donationbids?.filter((donobid: any) => donobid.donation === donation.pk);
+              const donobidsView = donobids?.map((donobid: any) => {
+                const bidObj = bids?.find((bid: any) => bid.pk === donobid.bid);
+                return (
+                  <tr>
+                    <td>{bidObj?.name}</td>
+                    <td>&euro;{(+donobid?.amount).toFixed(2)}</td>
+                    <td></td>
+                    <td>
+                      <button
+                        onClick={bidAction({
+                          pk: bidObj.pk,
+                          action: 'accept',
+                          state: 'OPENED',
+                        })}
+                        disabled={bidObj._internal?.saving}>
+                        Accept
+                      </button>
+                      <button
+                        onClick={bidAction({
+                          pk: bidObj.pk,
+                          action: 'deny',
+                          state: 'DENIED',
+                        })}
+                        disabled={bidObj._internal?.saving}>
+                        Deny
+                      </button>
```

Comment:
Queste cose non hanno feedback, ma di fatto dovrebbero andare a modificare l'oggetto bid. Sono le stesse cose che appaiono in "process pending bids".